app/recommended-books/systems-performance/SystemsPerformanceGuide.tsx

```
@@ -0,0 +1,1178 @@
+                    eBPFは「Linuxをプログラム可能なカーネルに変える」技術だとGregg自身が表現しており、eBPF
+                    Foundation（Linux
+                    Foundation傘下、Meta・Google・Microsoft・Netflix・Isovalent等が参画）を中心に、観測性だけでなくネットワーキング・セキュリティ領域でも標準的な基盤技術として採用が進んでいます。この生態系の中核をなす2つのフロントエンドが、単発の複雑なツールを書くのに向く<strong>BCC</strong>と、1行プログラムから手軽に始められる<strong>bpftrace</strong>です。
+                </p>
+                <h3 id="実践bpftraceの1行プログラム例" tabIndex={-1}>実践：bpftraceの1行プログラム例</h3>
+                <p>
+                    原著付録Cで多数紹介されている1行プログラムのうち、初学者が最初に試すのに適したものを抜粋します。
+                </p>
+                <ul>
+                    <li>
+                        execveによるプログラム実行をリアルタイムに表示（実行されるファイル名を表示）<br /><code
+                            >bpftrace -e 'tracepoint:syscalls:sys_enter_execve &#123; printf("%s\n",
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)